src/shared/components/case-editor/case-edit-page/case-edit-page.html

```diff
@@ -13,7 +13,8 @@ <h2 *ngIf="getCaseId()" class="heading-h2">
 
 <!-- Generic error heading and error message to be displayed only if there are no specific callback errors or warnings, or no error details -->
 <div *ngIf="error && !(error.callbackErrors || error.callbackWarnings || error.details)" class="error-summary"
-     role="group" aria-labelledby="edit-case-event_error-summary-heading" tabindex="-1">
+     role="status" aria-labelledby="edit-case-event_error-summary-heading" tabindex="-1">
```

Comment:
@phillip-whitaker - just a quick one... what's the implication of changing the `role` from `group` to `status`?

Reply:
I am not hugely sure, other than it is what the WCAG report suggested we did in the report. I think it changes the importance for screen readers essentially.

Reply:
OK, no worries - thanks.